velox/docs/monthly-updates/march-2022.rst

```diff
@@ -3,12 +3,12 @@ March 2022 Update
 ********************
 
 Documentation
--------------
+=============
```

Comment:
These need to be `sections` as well to match. It was not a problem previously since Sphinx automatically inferred some of this.